terraform/monitoring/00_provider.tf

```diff
@@ -27,7 +27,7 @@ provider "google" {
   version = ">=3.23.0"
 
   # credentials = "/path/to/creds.json"
-  # project = "project-id"
+  project = "${var.project_id}"
```

Comment:
I think Terraform doesn't like using quotes anymore. Can you try it without quotes?

Reply:
Yeah it actually doesn't like the ${} syntax anymore - kept the quotes but removed the interpolations to suppress the warnings. Will merge once tests pass unless there are other fixes.